### DISCUSSION TITLE
Scene of OBS Studio

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#3004] [General]
With the new version, we tried using the scenes created in OBS Studio with some cuts of the stage scenes (Main Stage, Speaker, Conductor, Chairperson, Table), but it became unfeasible as after displaying any image/video, the scene automatically returns to the main scene of OBS Studio, making it pointless to have created the scenes in OBS Studio. I see that in S-261 §.8, it refers to the number of cameras and says that one is sufficient to make a wide shot of the stage as per the example image. In this case, we understand that it refers to the number of cameras and not scenes, meaning one camera is enough for this procedure. Currently, it is difficult for the sound operator to keep changing every time media is presented. **I would like to know if it is possible to have an option for this scene change to be manual or automatic?** If not, it really doesn't make sense to have the scenes in OBS Studio. _I await your comments._

**Comment by mtdvlpr** [comment]
Thank you for reporting this! That's definitely a bug. After presenting the media, you should go back to the previously selected scene.

**Comment by mtdvlpr** [comment]
Continue in #3005
